src/kili/llm/services/export/dynamic.py

```diff
@@ -59,25 +60,30 @@ def export(
                         label["id"],
                         obfuscated_models,
                     )
+                    formatted_response = _format_json_response(
+                        json_interface["jobs"],
+                        round["annotations"],
+                        round["completion"],
+                        obfuscated_models,
+                    )
+                    label_data = {
+                        "author": label["author"]["email"],
+                        "created_at": label["createdAt"],
+                        "label_type": label["labelType"],
+                        "label": {},
+                    }
+                    if formatted_response["turn"]:
+                        label_data["label"]["turn"] = formatted_response["turn"]
+                    if step == total_rounds - 1 and formatted_response["asset"]:
```

Comment:
Did you double check with @stephnguyenkili if we wanted this information only for the last round?

If the main purpose for the customer is to handle each round as a separate result, we may need to export it on each rounds

Reply:
Check with @stephnguyenkili that's ok tu put on the last round